controllers/users.js

```diff
@@ -5,7 +5,6 @@ const logsQuery = require("../models/logs");
 const imageService = require("../services/imageService");
 const { profileDiffStatus } = require("../constants/profileDiff");
 const { logType } = require("../constants/logs");
-
```

Comment:
I think this shouldn't be part of this PR. You might have formatted accidentally.

Reply:
ok, i added the space back